integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java

```diff
@@ -1,17 +1,25 @@
-package com.sequenceiq.it.cloudbreak.newway;
+package com.sequenceiq.it.cloudbreak.newway.entity.ldap;
 
 import java.util.function.BiConsumer;
 import java.util.function.Function;
 
 import com.sequenceiq.it.IntegrationTestContext;
+import com.sequenceiq.it.cloudbreak.newway.Assertion;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.GherkinTest;
+import com.sequenceiq.it.cloudbreak.newway.Prototype;
+import com.sequenceiq.it.cloudbreak.newway.ResourceAction;
+import com.sequenceiq.it.cloudbreak.newway.TestParameter;
 import com.sequenceiq.it.cloudbreak.newway.action.Action;
+import com.sequenceiq.it.cloudbreak.newway.action.ldap.LdapConfigDeleteAction;
 import com.sequenceiq.it.cloudbreak.newway.action.ldap.LdapConfigPostAction;
 import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
 import com.sequenceiq.it.cloudbreak.newway.v4.LdapConfigV4Action;
 
 @Prototype
 public class LdapConfig extends LdapConfigEntity {
-    private static final String LDAPCONFIG = "LdapCONFIG";
+
```

Comment:
pls, change to the new conception (TestDto/TestAction..)

Reply:
I cannot use the new naming convention till we do not agree on refactoring the whole int test code base. I won't rename LdapConfig to LdapConfigTestService and make it extend the LdapConfigTestDto. And if it does no extend the Dto, then the tests won't start, so there is need for a bigger rework here.